lib/android/app/src/main/java/com/reactnativenavigation/parse/NavigationOptions.java

```diff
@@ -39,6 +45,13 @@ public static NavigationOptions parse(JSONObject json) {
 		result.topBarHidden = BooleanOptions.parse(json.optString("topBarHidden"));
 		result.animateTopBarHide = BooleanOptions.parse(json.optString("animateTopBarHide"));
 
+		try {
+//			result.leftButtons = json.getJSONArray("leftButtons");
+			result.rightButtons =  Button.parseJsonArray(json.optJSONArray("rightButtons"));
+		} catch (JSONException e) {
```

Comment:
I'm not sure throwing an Exception is the best way. It better just to use `opt()` and check for null.

Reply:
Completely agree, was mostly like that whilst I was developing.